pkg/serverless/trace/span_modifer_test.go

```diff
@@ -118,12 +118,15 @@ func TestLambdaSpanChan(t *testing.T) {
 		Source:        agnt.Receiver.Stats.GetTagStats(info.Tags{}),
 	})
 	timeout := time.After(2 * time.Second)
+	receivedSpanCount := 0
 	var span *pb.Span
 	select {
 	case ss := <-lambdaSpanChan:
+		receivedSpanCount++
 		span = ss
 	case <-timeout:
 		t.Fatal("timed out")
 	}
 	assert.Equal(t, "myTestService", span.Service)
+	assert.Equal(t, receivedSpanCount, 1)
```

Comment:
It looks like this value will only ever be 1 or 0, yeah? If it's 0, then the above assert will fail because `span` would be nil.

If the value _could_ also be 2, then this would make the assertion more interesting/helpful.

Reply:
This change is in response to our discussion yesterday. If we remove the change to `span_modifier` and then run this test, it will fail (as the value would be > 1)